### ISSUE TITLE
Text with no height should be considered invisible and therefore should not be tested for color contrast

### ISSUE THREAD

**Issue by jongund** [opena11y/oaa-rules#44]
Hi Jon,
I noticed that the OAA tool, and therefor library is flagging an element as being in violation of contrast on our site.  That element's text is visually hidden and is only provided for screen reader use.  If the text were not visually hidden with CSS, then yes, it would be a problem.

.wdn-text-hidden { font-size: 0px; width: 1px; height: 1px; display: inline-block; overflow: hidden; position: absolute !important; border: 0px none !important; padding: 0px !important; margin: 0px !important; clip: rect(1px, 1px, 1px, 1px); }
Other accessibility checkers seem to check if the font-size is greater than 0px when checking for contrast.
Do you see this as something that needs to be fixed in OAA, or do you see an actual accessibility issue here?
Thanks,
Michael Fairchild

**Comment by jongund** [comment]
Algorithm to identify hidden content was updated to include testing for height of content.  Content less than 4 pixels high is considered invisible visually for the purpose of color contrast tests
